test_elasticsearch/run_tests.py

```diff
@@ -21,10 +21,10 @@ def fetch_es_repo():
 
     # no repo
     if not exists(repo_path) or not exists(join(repo_path, ".git")):
-        print("No elasticsearch repo found...")
-        # set YAML DIR to empty to skip yaml tests
-        environ["TEST_ES_YAML_DIR"] = ""
-        return
```

Comment:
this option was useful to avoid running the rest tests as they take a long time, this will mean that a typo in a setting will cause the entire ES repo to be cloned which is a very expensive process

Reply:
That's true, I'll add back a way to disable running API tests. Will also apply that flag onto the `oss` Jenkins CI config.

Reply:
the rest tests that are running now are for the `oss` release